src/verify_ce_config.py

```diff
@@ -10,86 +10,135 @@
 
 
 def error(msg):
+    """Exit with error 'msg'
+    """
     sys.exit("ERROR: " + msg)
 
 
 def warn(msg):
+    """Print warning 'msg'
+    """
     print "WARNING: " + msg
 
 
+def parsed_route_names(entries_config):
+    """Return names of job routes that can be parsed as proper ClassAds
+    """
+    return [x['Name'] for x in classad.parseAds(entries_config)]
```

Comment:
You might want to do something with whatever exceptions classad.parseAds() might throw on unparseable ads.

Reply:
Unfortunately, strings that have unparseable ads in them don't raise any sort of exception. The unparseable ads are just ignored.

However, `ValueError` is raised if you pass `classad.parseAds` a non-string value so we could catch that but I'm not sure what the proper behavior would be other than re-raising the exception.

Reply:
It should be OK to catch it elsewhere in the program -- I'd just like to make sure there's a nice error message instead of a stack trace.